src/Microsoft.Health.Dicom.Core/Exceptions/ElementValidationExceptionFactory.cs

```diff
@@ -1,115 +0,0 @@
-﻿// -------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-// -------------------------------------------------------------------------------------------------
-
-using System.Globalization;
-using EnsureThat;
-using FellowOakDicom;
-using Microsoft.Health.Dicom.Core.Features.Validation;
-
-namespace Microsoft.Health.Dicom.Core.Exceptions
-{
-    internal static class ElementValidationExceptionFactory
```

Comment:
Most of these methods were only called once, so I think it makes sense to just create their respective exceptions in-line as opposed to delegating to a factory